.env

```diff
@@ -0,0 +1 @@
+DBPASSWORD=datadriver101
```

Comment:
you need to add `.env` to your `.gitignore`
then you can also manually delete this file from github